cmd/cluster-network-operator/main.go

```diff
@@ -25,7 +23,7 @@ import (
 func printVersion() {
 	log.Printf("Go Version: %s", runtime.Version())
 	log.Printf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
-	log.Printf("operator-sdk Version: %v", sdkVersion.Version)
+	log.Printf("operator-sdk Version: %v", Version)
```

Comment:
I don't think there's any point in logging this any more

Reply:
yeah, just left it in case *something* was consuming it...